### DISCUSSION TITLE
SmartCards support

### DISCUSSION THREAD

**Discussion by pando85** [kanidm/kanidm#2725] [General]
Hello guys,

I was reading this https://fy.blackhats.net.au/blog/2024-04-26-passkeys-a-shattered-dream/ and I was wondering if in the future it will be possible to use any smartcard or browser certificates as an authentication method.

I was thinking about using the Spanish government ID which contains a certificate that you uses typically for your government-related procedures. 

How do you see it? Is it Kanidm going in to that direction?

**Comment by Firstyear** [comment]
https://github.com/kanidm/kanidm/issues/2417

I don't see why you couldn't use those spanish id cards, but I'd need to know what's in the certificate to know how to map it.
